autonomy/cli/helpers/chain.py

```diff
@@ -199,20 +200,21 @@ def mint_component(  # pylint: disable=too-many-arguments, too-many-locals
             f"Please provide hash for NFT image to mint component on `{chain_type.value}` chain"
         )
 
-    try:
-        verify_component_dependencies(
-            ledger_api=ledger_api,
-            contract_address=ContractConfigs.get(
-                COMPONENT_REGISTRY_CONTRACT.name
-            ).contracts[chain_type],
-            dependencies=dependencies,
-            package_configuration=package_configuration,
-            skip_hash_check=skip_hash_check,
-        )
-    except FailedToRetrieveComponentMetadata as e:
-        raise click.ClickException(f"Dependency verification failed; {e}") from e
-    except DependencyError as e:
-        raise click.ClickException(f"Dependency verification failed; {e}") from e
+    if not skip_dependencies_check and chain_type == ChainType.ETHEREUM:
```

Comment:
Do we want to also do 
```suggestion
    if not skip_dependencies_check and (chain_type == ChainType.ETHEREUM or chain_type == ChainType.GOERLI):
```